go.mod

```diff
@@ -1,26 +1,23 @@
 module github.com/mvmaasakkers/certificates
 
-go 1.13
+go 1.18
 
 require (
-	cloud.google.com/go v0.57.0 // indirect
-	git.apache.org/thrift.git v0.12.0 // indirect
```

Comment:
Note that this may _not_ actually fix #25 because this is only getting pruned from `go.mod` due to being an `// indirect` dep and the new `go` `1.17` module graph pruning of `go.mod`.

So if one of the remaining things mentioned in `go.mod` still pulls this in under the covers, then it could still break. 

One way to fix that is to explicitly bump this to `v0.13.0`, but there is the risk of backwards-breaking changes, etc... 

The best way to fix this actually would be to figure out _which_ direct dep is pulling this indirectly and then bump that direct dep to a version that pulls in a newer `thrift` version.